### DISCUSSION TITLE
Welcome to mantis Discussions!

### DISCUSSION THREAD

**Discussion by santanusinha** [PhonePe/mantis#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 0xbharath** [comment]
Hi folks,

This is Bharath. I'm one of the developers of Mantis. My area of interests include AppSec and Attack Surface Management (ASM).

As part of Mantis, I helped with the ASM flows and installation scripts. 

Happy to help folks with Mantis and discuss new ideas!

**Comment by Prateek-Thakare** [comment]
Hey there!!
I am Prateek. I'm also one of the developers of Mantis. Feel free to request any feature or raise any issue. I'll be able to help you with new tool integrations and anything related to Mantis.

**Comment by wisdomfreak-1** [comment]
Hi folks,

This is Saddam. I'm also one of the developer of Mantis. I'll be able to help you with Dashboard, tools integrations.

Happy to help folks with Mantis and discuss new ideas!

**Comment by vancong160202** [comment]


**Reply by 0xbharath** [reply]
@vancong160202 Mantis is intended to be an Attack Surface Management framework rather than a web app vulnerability scanner. All the tools we have included so far help enumerating attack surface, finding low hanging security issues and known security issues. 

However, it is relatively easy to integrate tools into Mantis for your own workflows and make it work similar to a web app vulnerability scanner.

**Reply by vancong160202** [reply]
Can you guide me to integrate tools into Mantis to scan for web vulnerabilities such as sql injection, xss...? I think when integrated, Mantis will have more applications and will be a useful framework

**Reply by 0xbharath** [reply]
@vancong160202 Thank you for considering contributing to Mantis. 

@Prateek-Thakare would be of better help here regarding code contributions.

Ideally, can you start a discussion under "Contribution" regarding this https://github.com/PhonePe/mantis/discussions/categories/contribution

Try to mention the following in the discussion:

- Tool(s) you'd like to see integrated
- What will these tool(s) find and how would they help

**Reply by Prateek-Thakare** [reply]
@vancong160202 If you want to integrate tools you can refer integration doc: https://phonepe.github.io/mantis/new-scanner-integrations/new-scanner-integration.html. Feel free to reach out if you face any difficulties.
